simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -63,19 +68,19 @@ export const HowToCreate = () => (
 
 export const HowToJoin = () => (
   <div className={styles['join-info']}>
-    <h2>Join a queue without being there</h2>
+    <h2>How do people join a queue?</h2>
     <div className={styles['queue-container']}>
       <div className={styles['queue']}>
         <ul className={styles['list-info']}>
           <img src="/images/mobile_friendly-24px.svg" alt="Mobile friendly" />
           <li>
-            <h3>Join queue</h3>
-            <p>Get queue link or enter queue name to join queue from anywhere</p>
+            <h3>Join a queue</h3>
+            <p>Get the shared queue link and enter your details to join a queue</p>
           </li>
           <img src="/images/alarm-24px.svg" alt="Alarm" />
           <li>
-            <h3>Wait for your turn</h3>
-            <p>Check you status in queue. Get notified when your turn is up</p>
+            <h3>Waiting for your turn</h3>
+            <p>Check your token status in queue and get notified when your turn is up</p>
           </li>
```

Comment:
Maybe stick to wait for your turn since we're still using join a queue and not joining.